tools/cbmc/proofs/HTTP/global_state_HTTP.c

```diff
@@ -246,9 +248,21 @@ IotHttpsResponseHandle_t allocate_IotResponseHandle() {
     safeMalloc(sizeof(struct _httpsResponse));
   if(pResponseHandle) {
     size_t headerLen;
-    size_t bodyLen;
+    __CPROVER_assume(headerLen < CBMC_MAX_OBJECT_SIZE);
     pResponseHandle->pHeaders = safeMalloc(headerLen);
+    pResponseHandle->pHeadersEnd = pResponseHandle->pHeadersCur = pResponseHandle->pHeaders;
+    if (pResponseHandle->pHeaders) {
+      pResponseHandle->pHeadersEnd = pResponseHandle->pHeaders + headerLen - 1;
+    }
+
+    size_t bodyLen;
+    __CPROVER_assume(bodyLen < CBMC_MAX_OBJECT_SIZE);
     pResponseHandle->pBody = safeMalloc(bodyLen);
+    pResponseHandle->pBodyEnd = pResponseHandle->pBodyCur = pResponseHandle->pBody;
+    if (pResponseHandle->pBody) {
+      pResponseHandle->pBodyEnd = pResponseHandle->pBody + bodyLen - 1;
+    }
+
```

Comment:
Comments for the header apply here to the body.

Reply:
As per our discussion, I have now added offsets that are non-deterministically assigned between 0 and `bodyLen`, and are then added to `pBody`.